presidio_grpc_wrapper/server.py

```diff
@@ -70,12 +70,25 @@ def Anonymize(self, request, context):
 
     def AnalyzeAndAnonymize(self, request, context):
```

Comment:
This function is incorrect as it's missing the anonymize options. If you look at the Anonymize function you'll see the options being passed in.